packages/three-instanced-uniforms-mesh/src/InstancedUniformsMesh.js

```diff
@@ -143,7 +147,8 @@ function getItemSizeForValue (value) {
     : typeof value === 'number' ? 1
     : value.isVector2 ? 2
     : value.isVector3 || value.isColor ? 3
-    : value.isVector4 ? 4
+    : value.isVector4 || value.isQuaternion ? 4
+    : value?.elements ? value.elements.length
```

Comment:
Can you remove both instances of the nullish coalescing `?.` in this file? They're redundant here (the code would have already errored if value was nullish) and it tends to add verbose code when transpiled.

Reply:
done